packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -253,18 +265,6 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
           }),
       },
 
-      ':last-child': {
```

Comment:
Why was this removed? It's really hard to guess what the changes are as the screener differences are already accepted.. Can you provide some screenshot of why this is not needed anymore?

Reply:
This block of styles only applied to vertical menu items which had new set of styles applied.
The vertical menu items don't have border radius any longer, radius is applied to parent "Menu" element instead.